.travis.yml

```diff
@@ -13,6 +13,7 @@ language: python
 python:
  - 3.6
  - 3.7
+ - 3.8
 
```

Comment:
You can probably add 3.9 here now, right?

Reply:
3.9 isn't [due out until October](https://www.python.org/dev/peps/pep-0596/#schedule).

We should probably add 3.9-dev to Travis at some point. Is Beta1 a good time to do that in a new Python's release cycle?

Reply:
Ah, I saw it on my PATH and assumed it was the final version. Seems Fedora packages up betas.

We can leave this til release, I suspect, since python's API is pretty stable and we've no history yet of breaking between minor versions